include/z64player.h

```diff
@@ -549,6 +549,21 @@ typedef enum {
     /* 4 */ PLAYER_LEDGE_CLIMB_4
 } PlayerLedgeClimbType;
 
+typedef enum {
+    /* 0 */ PLAYER_KNOCKBACK_NONE,
+    /* 1 */ PLAYER_KNOCKBACK_SMALL,
+    /* 2 */ PLAYER_KNOCKBACK_LARGE,
+    /* 3 */ PLAYER_KNOCKBACK_LARGE_SHOCK
+} PlayerKnockbackType;
+
+typedef enum {
+    /* 0 */ PLAYER_DAMAGE_RESPONSE_NONE,
+    /* 1 */ PLAYER_DAMAGE_RESPONSE_KNOCKBACK_LARGE,
+    /* 2 */ PLAYER_DAMAGE_RESPONSE_KNOCKBACK_SMALL,
+    /* 3 */ PLAYER_DAMAGE_RESPONSE_ICE_TRAP,
+    /* 4 */ PLAYER_DAMAGE_RESPONSE_ELECTRIC_SHOCK
+} PlayerDamageResponseType;
```

Comment:
For a while ive been meaning to call this "damage reaction". I think both words convey the same thing so its a matter of preference. The term "damage reaction" was used in the gamecube titles to mean something similar, but thats a small point.

Really, I think "damage response/reaction" would be a good term for what we currently call "damage effect" in colliders. So using it here for this player specific thing may hurt if we wanted to use that terminology with colliders. But I wouldnt know what to call this value in that case. they really mean the same thing

Reply:
Can you elaborate on

> "damage response/reaction" would be a good term for what we currently call "damage effect" in colliders

Which fields specifically are you referring to?

Reply:
as discussed on discord, I think its best if we revert these name changes for now and consider these names later at a broader scale with the at/acHitEffect fields in colliders.

Reply:
https://discord.com/channels/688807550715560050/838852326231769089/1214214286340128869
to
https://discord.com/channels/688807550715560050/838852326231769089/1214230605353586748

Reply:
#1925

Reply:
#1925 (currently) names `this->actor.colChkInfo.acHitEffect` -> `this->actor.colChkInfo.playerACHitReaction` / `PlayerACHitReaction` / `PLAYER_AC_HIT_REACTION_`

`func_808382DC` as highlighted by fig above maps `PLAYER_AC_HIT_REACTION_` values to this PR's current `PLAYER_DAMAGE_RESPONSE_`

Reply:
apologies for sitting on this. I've looked through pr #1925 a few times, I've just been struggling to get a definitive picture in my head of how things should be so I haven't commented on it.

Perhaps I can sidestep the collision_check discussion for now by renaming `PLAYER_DAMAGE_RESPONSE_` to `PLAYER_HIT_RESPONSE_` instead.